### DISCUSSION TITLE
Lesson 4: ValueError: {'code': -32000, 'message': 'err: max fee per gas less than block base fee: address 0x51b4aFd0e28f3CEcdE15e9fb348327A7e232B63e, maxFeePerGas: 2466968479 baseFee: 2476954592 (supplied gas 201271279)'}

### DISCUSSION THREAD

**Discussion by Aishwaria8786** [smartcontractkit/full-blockchain-solidity-course-py#1867] [Q&A]
I used Alchemy. It is showing this error.
```ValueError: {'code': -32000, 'message': 'err: max fee per gas less than block base fee: address 0x51b4aFd0e28f3CEcdE15e9fb348327A7e232B63e, maxFeePerGas: 2466968479 baseFee: 2476954592 (supplied gas 201271279)'}```
What should I do now?

**Comment by Niferu** [comment]
Hello @Aishwaria8786 

Why didn't you post code that is generating such error? Without that at least I cannot help...

**Reply by cromewar** [reply]
@Neftyr  Is right please do share the code.
